centreon-open-tickets/packaging/centreon-open-tickets.spectemplate

```diff
@@ -9,7 +9,7 @@
 
 Name:           centreon-open-tickets
 Version:        %{version}
-Release:        %{release}%{?dist}
+Release:        %{release}
```

Comment:
Shouldn't we modify the line 35 too : %setup -q -n %{name}-%{version}

Reply:
indeed, fixed